optaplanner-benchmark/pom.xml

```diff
@@ -106,6 +106,12 @@
       <groupId>org.freemarker</groupId>
       <artifactId>freemarker</artifactId>
     </dependency>
+    <!-- For (deprecated) JavaScript support. -->
+    <dependency>
+      <groupId>org.mozilla</groupId>
+      <artifactId>rhino</artifactId>
+      <scope>optional</scope>
```

Comment:
There is no scope optional.
It's:
```
<optional>true</optional>
```

No idea why maven isn't fail fasting when you do this.

Reply:
If you ran any tests from a client's perspective (say in optaplanner-examples) on JDK 11 they probably had rhino in the classpath...